build-scripts/ubuntu-1604/build-3rd-parties.sh

```diff
@@ -100,7 +100,7 @@ build_from_pypi python-dateutil 2.6.1
 build_from_pypi semver 2.7.9
 build_from_pypi pygments 2.2.0
 build_from_pypi psutil 5.6.6
-build_from_pypi pyzmq 18.1.0 bundled
+build_from_pypi pyzmq 22.3.0 --zmq=bundled
```

Comment:
Do we want to upgrade `pyzmq` for Ubuntu `16.04` and `20.04`?

Reply:
Yes, though I'm holding off on making the changes to the master branch for the moment.